src/main/java/uk/gov/hmcts/probate/service/NotificationService.java

```diff
@@ -34,8 +37,11 @@
     private final NotificationClient notificationClient;
     private final MarkdownTransformationService markdownTransformationService;
     private final PDFManagementService pdfManagementService;
+    private static final String DOC_SUBTYPE = "will";
```

Comment:
Consider add constants to teh constants file under model.

Reply:
👍